sys/canpb/include/canpb/CanProtobuf.hpp

```diff
@@ -0,0 +1,115 @@
+// Copyright (C) 2021 COGIP Robotics association <cogip35@gmail.com>
+// This file is subject to the terms and conditions of the GNU Lesser
+// General Public License v2.1. See the file LICENSE in the top level
+// directory for more details.
+
+/// @ingroup     sys_canpb
+/// @brief       Exchange Protobuf messages over CAN module.
+/// @details     This module provides a class that controls an CAN to send and
+///              receive Protobuf messages using EmbbededProto.
+///              Each library/application using canpb must define a unique id (32-bit integer
+///              of type cogip::canpb::uuid_t) and registrer a callback function to send
+///              or receive their own messages.
+///              See `examples/canb`.
+/// @{
+/// @file
+/// @author      Eric Courtois <eric.courtois@gmail.com>
```

Comment:
Not me, not alone ;-)
Same for other headers.

Reply:
Done